recipes/wineditline/all/test_package/CMakeLists.txt

```diff
@@ -1,9 +1,6 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.15)
 project(test_package C)
 
-include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
-conan_basic_setup(TARGETS)
-
 find_package(wineditline REQUIRED CONFIG)
 
 add_executable(test_package test_package.c)
```

Comment:
(refers to line 7) if original cmakelists is to be used:
`target_link_libraries(test_package PRIVATE wineditline::edit)`

in addition with `self.cpp_info.components["edit"].libs = ["edit_static"]` in recipe package_info()